### DISCUSSION TITLE
Trigger of HRR curve based on certain temperature

### DISCUSSION THREAD

**Discussion by MohamedSalah989** [firemodels/fds#12811] [Q&A]
Hi FDS team,
How can I identify a trigger temperature control condition for the pre-discribed HRR curve for a material in the Pyrosim FDS models or the ignition temperature have the same function?

Regards
Mohamed Salah
![Trigger of HRR curve](https://github.com/firemodels/fds/assets/138606386/a03f5e71-cfe7-4504-9fd7-5aad9f26277a)

**Comment by drjfloyd** [comment]
See the FDS User's Guide chapter on controls.  

For help with the Pyrosim interface for FDS see the Pyrosim user guide or contact Thunderhead Engineering.
